### PR TITLE
[ZRH-2020] Open registration

### PR DESCRIPTION
**Pull request by RomanoRoth**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by mattstratton** [comment]
> In the **_2020-zurich.yml_**, please add **start** and **end** dates for registration.
> 
> registration_date_start: # start accepting registration. Leave blank if registration is not open yet
> registration_date_end: # close registration. Leave blank if registration is not open yet.

These fields are optional and don't need to be set if registration isn't available yet.

**Comment by mattstratton** [comment]
whoops - I see that this is setting registration to be open. If you want the "register" button to show up on the main event page, you need to set these fields.

**Comment by mattstratton** [comment]
to be clear, if you want this button to appear on your `welcome` page, you need to set those fields:

image

**Comment by mattstratton** [comment]
For reference, this is the code that uses those fields:

https://github.com/devopsdays/devopsdays-web/blob/master/themes/devopsdays-theme/layouts/partials/events/cta.html#L22-L47

**Comment by mattstratton** [comment]
to be even more clear; if you want the button to show up, you have to set `registration_date_start`. But `registration_date_end` is optional and can be left off.

**Comment by mattstratton** [comment]
I set `registration_date_start` to today; let's see the deploy preview and make sure that works :)

**Comment by RomanoRoth** [comment]
@mattstratton thanks. I've set the dates now.

**Comment by RomanoRoth** [comment]
@MikeRosTX I've set the dates now.

File: content/events/2020-zurich/registration.md [hunk 1]
```diff
@@ -4,8 +4,14 @@ Type = "event"
...
+<h3>Registration for DevOpsDays Zürich 2020 is open.</h3> 
 
-Embed registration iframe/link/etc.
-</div></div>
+<div style="width:100%; text-align:left;">
+<link rel="stylesheet" type="text/css" href='https://css.tito.io/v1.1' />
+<script src='https://js.tito.io/v1' async></script>
+<tito-widget event="dod/dodzh2020"></tito-widget>
 </div>
+<br/ß>
```
Comment (mattstratton) [inline content/events/2020-zurich/registration.md]: Aha! This is causing some html errors. I'll fix it.

Comment (RomanoRoth) [reply content/events/2020-zurich/registration.md]: thanks!

**Comment by mattstratton** [comment]
looks good to me now. i'll merge it. thanks everyone for your efforts!

**Comment by RomanoRoth** [comment]
Thank you very much!

**Pull request merged**